src/sass/components/_cta.scss

```
@@ -1,158 +1,256 @@
+
+	@media (orientation: landscape) {
+		display: block;
+		position: relative;
+	}
+}
+
+#cta-dots {
+	display: none;
+
+	@media (orientation: landscape) {
+		display: block;
```

Stylelint (17.6.0)

[error] Unexpected duplicate "display" (declaration-block-no-duplicate-properties)

(declaration-block-no-duplicate-properties)